projects/hostap/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "<your_project_homepage>"
+primary_contact: "<primary_contact_email>"
+auto_ccs:
+   - "TODO"
+sanitizers:
+  - address
+  - undefined
+  - memory
+fuzzing_engines:
```

Comment:
You can remove lines 9-10 and the project will be fuzzed by both AFL and libFuzzer fuzzing engines. If you know that AFL is not going to work, then it's fine to enable libFuzzer only.